2-planning/3-documentation.adoc

```diff
@@ -0,0 +1,134 @@
+= Reference Documentation
+
+So, you handed out the mock server to the various API client teams, and thought
+they would get straight to work trying to integrate it. Sadly your inbox is now
+rammed full of questions from those client developers asking how the heck they
+use the API.
+
+*REST devs:* You don't need reference docs, the message is self documenting
+through the rules of HTTP and HATEOAS explains interactions!
+
+*GraphQL devs:* You don't need reference docs, you have GraphiQL!
+
+*gRPC devs:* You don't need reference docs, you have .proto files!
+
+Guess what? If you want that sweet-sweet feedback, you'll need to help your
+clients implement the API, and the best way to avoid inboxmageddon is to
```

Comment:
`inboxmageddon` 😂 👌stealing that word, adding `slackmageddon` next

Reply:
:D